src/components/organisms/AppRouter/AppRouter.jsx

```diff
@@ -81,7 +81,15 @@ const AppRouter = ({ defaultVenue }) => {
         />
         <Route
           path="/"
-          component={() => <Redirect to={venueLandingUrl(defaultVenue)} />}
+          component={() => (
+            <Redirect
+              to={
+                defaultVenue === "playa"
+                  ? venueInsideUrl(defaultVenue)
```

Comment:
I think Chris did that already: https://github.com/sparkletown/sparkle/pull/329

Reply:
it was venueLandingUrl(defaultVenue) which sends you to "v/playa". For the playa we need to go to "in/playa" with the venueInsideUrl(defaultVenue).